File: java/oop/InterfaceApp/src/com/techlabs/icrudable/CustomerDB.java
package com.techlabs.icrudable;

public class CustomerDB implements ICRUDable {
	public void create(){
		System.out.println("CustomerDB is created");
	}
	public void update(){
		System.out.println("CustomerDB is updated");
	}
	public void read(){
		System.out.println("CustomerDB is Showing");
	}
	public void delete(){
		System.out.println("CustomerDB is deleting");
	}
}

File: dotNet/CSharp/InterFaceCRUDApp/InterFaceCRUDApp/CustomerDB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InterFaceCRUDApp
{
    public class CustomerDB : ICRUDAble
    {
        public void Create()
        {
            Console.WriteLine("CustomerDB is Created");
        }

        public void Delete()
        {
            Console.WriteLine("CustomerDB is Deleting");
        }

        public void Read()
        {
            Console.WriteLine("CustomerDB is Reading");
        }

        public void Update()
        {
            Console.WriteLine("CustomerDB is Updated");
        }
    }
}
